src/Native/configure.cmake

```diff
@@ -234,6 +234,22 @@ check_cxx_source_compiles(
 
 check_cxx_source_compiles(
     "
+    #include <netinet/tcp.h>
+    int main() { int x = TCP_ESTABLISHED; return x; }
+    "
+    HAVE_TCP_H_TCPSTATE_ENUM
+)
+
+check_cxx_source_compiles(
+    "
+    #include <netinet/tcp_fsm.h>
+    int main() { int x = TCPS_ESTABLISHED; return x; }
+    "
+    HAVE_TCP_FSM_H
+)
+
```

Comment:
Ah I see, I think that would work as well. I don't really have a strong preference, but I think mine is a little clearer since I'm not actually interested in the size of the type. I checked the CMake docs and it mentioned either one could be used.